api/__init__.py

```diff
@@ -29,6 +33,12 @@
 dotenv_path = join(dirname(__file__), '.env')
 load_dotenv(dotenv_path)
 
+# Import for env file:
```

Comment:
This looks redundant with the block above.

Reply:
This is indeed redundant, I will remove